### DISCUSSION TITLE
How to create a dmg file?

### DISCUSSION THREAD

**Discussion by NAKAK10** [flet-dev/flet#900] [Q&A]
Hi 👋
I'm trying to make a distributable desktop app.

 🙏

**Comment by FeodorFitsner** [comment]
Some useful links:
https://www.wikihow.com/Make-a-DMG-File-on-a-Mac
https://apple.stackexchange.com/questions/60029/how-to-create-a-disk-image-from-the-command-line
https://github.com/create-dmg/create-dmg
